DOCS/xml-fields.md

```diff
@@ -18,17 +18,15 @@ The XML template for each license is stored in a separate file in the [`src/` di
 
 For license exceptions, the same pattern is used, with exceptions stored in the [`src/exceptions/` directory](../src/exceptions).
 
-A corresponding "test text" file, with a plain text version of the license or exception, should be added to the [`test/simpleTestForGenerator/` directory](../test/simpleTestForGenerator). Unlike the XML files, the same directory is used for the test files for licenses as well as exceptions. This text file will be used by the CI system to confirm that the XML template actually matches the "test text" for each license and exception.
+A corresponding "test text" file, a plain text version of the license or exception, is stored in the [`test/simpleTestForGenerator/` directory](../test/simpleTestForGenerator). Unlike the XML files, the same directory is used for the test files for licenses as well as exceptions. This text file will be used by the CI system to confirm that the XML template actually matches the "test text" for each license and exception.
 
 ### Whitespace and formatting
 
-We aren't picky about tabs vs. spaces, number of spaces for indenting, etc.
-
-At some point maybe we'll pick a standard and update the existing files, but we haven't yet, so use whatever you like.
+We aren't picky about tabs vs. spaces, number of spaces for indenting, etc. (And note, whitespace is not relevant for matching license text.) That being said, if you use the [SPDX Online Tools-license submission](https://tools.spdx.org/app/license_requests/) to create the XML and TXT files, there is a "beautify" option that formats the file nicely. 
```

Comment:
The link here goes to the list of submitted licenses, rather than the form for submitting a license -- I think the URL should be https://tools.spdx.org/app/submit_new_license/ instead